src/pages/resources/[resourceSlug].js

```diff
@@ -54,6 +55,7 @@ const ResourcePage = ({ propResource, propTheme, previewMode }) => {
       { theme && resource
         ? (
           <>
+            <Head title={resource.title} ogImage={resource.image ? resource.image.filename : '/logo.png'} />
```

Comment:
Maybe we could set the `logo.png` fallback in your `Head` component instead? And then allow null/blank image to be passed.

Reply:
Done!